docs/sphinx/reference-libobs-graphics-effects.rst

```diff
@@ -154,6 +154,36 @@ HLSL format.
 
 ---------------------
 
+.. function:: size_t gs_param_get_num_annotations(const gs_eparam_t *param)
+
+   Gets the number of annotations associated with the parameter.
+
+   :param param: Param object
```

Comment:
Style:  Align "Param object" with with the lines below.  See examples throughout file.